app/controllers/dashboard/dashboard-activity-controller.js

```diff
@@ -171,3 +177,35 @@ function getTransactionDateRange (period) {
 
   return { fromDateTime, toDateTime }
 }
+
+async function getStripeAccountDetails (paymentProvider, gatewayAccountId, correlationId) {
+  if (paymentProvider === 'stripe') {
+    try {
+      const stripeResponse = await connector.getStripeAccount(gatewayAccountId, correlationId)
+      const stripeAccoundId = stripeResponse.stripeAccountId
+
+      try {
+        const fullStripeAccountDetails = await retrieveAccountDetails(stripeAccoundId)
+
+        const reducedStripeAccountDetails = {
+          payouts_enabled: fullStripeAccountDetails.payouts_enabled,
+          requirements: null
```

Comment:
I don't think this needs to be set to null

Reply:
Done